test/views/CalculationViewSpec.scala

```diff
@@ -107,6 +108,10 @@ class CalculationViewSpec extends SpecBase with ViewBehaviours with ViewUtils {
           doc.select(Selector.breadcrumbWithLink(2)).attr("href") shouldBe appConfig.vatOverviewUrl
           doc.select(Selector.breadcrumbWithLink(3)).attr("href") shouldBe controllers.routes.IndexController.onPageLoad.url
         }
+
+        "not display TTP content" in {
+          doc.select(".ttp-content").isEmpty shouldBe true
+        }
```

Comment:
Wondering if it's worth adding this test to where the penalty is _not_ estimated?